src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/PublicationController.cs

```diff
@@ -1,29 +1,34 @@
-﻿using System.Net.Mime;
+﻿#nullable enable
+using System.Net.Mime;
 using System.Threading.Tasks;
 using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces;
-using GovUk.Education.ExploreEducationStatistics.Publisher.Model.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
 using Microsoft.AspNetCore.Mvc;
-using static GovUk.Education.ExploreEducationStatistics.Common.Services.FileStoragePathUtils;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Controllers
 {
     [Route("api")]
     [Produces(MediaTypeNames.Application.Json)]
     public class PublicationController : ControllerBase
     {
-        private readonly IFileStorageService _fileStorageService;
+        private readonly IPublicationService _publicationService;
 
-        public PublicationController(IFileStorageService fileStorageService)
+        public PublicationController(IPublicationService publicationService)
         {
-            _fileStorageService = fileStorageService;
+            _publicationService = publicationService;
         }
 
         [HttpGet("publications/{slug}/title")]
         public async Task<ActionResult<PublicationTitleViewModel>> GetPublicationTitle(string slug)
         {
-            return await _fileStorageService
```

Comment:
Not sure if it's related to these changes or not but `PublicContentPublicationTitlePath` in `FileStoragePathUtils` has been left unused.

Reply:
It is. I originally intended to create a new cache file in additiona to `publication.json`: `publication-title.json`. But in the end I went back on that decision and the publication title is fetched from `publication.json`.